testing/trino-tests/src/test/java/io/trino/security/TestAccessControl.java

```diff
@@ -158,15 +158,15 @@ protected SystemAccessControl delegate()
                 })
                 .withGetViews((connectorSession, prefix) -> {
                     ConnectorViewDefinition definitionRunAsDefiner = new ConnectorViewDefinition(
-                            "select 1",
+                            "select 1 as test",
```

Comment:
Why? separate commit?

Reply:
It wont work without these changes so i think it can be in the same commit

Reply:
this looks MV unrelated, so it looks unrelated to commit title, right?

Reply:
Please tell me if current version is better